unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarName.cs

```diff
@@ -32,6 +33,16 @@ public void SetName(string name)
         }
     }
 
+    void OnDisable()
+    {
+        SetTalking(false);
+    }
+
+    public void SetTalking(bool talking)
+    {
+        talkingIconContainer.SetActive(talking);
```

Comment:
As this is called from `OnDisable` that can be called in weird contexts (i.e. when the object is being destroyed), I'd add a null check just in case.

Reply:
Done!